File: src/main/resources/static/js/service/labrecordService.js
//服务层
app.service('labrecordService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../labrecord/findAll');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../labrecord/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../labrecord/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../labrecord/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../labrecord/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../labrecord/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../labrecord/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/com/ty/lab/service/LabrecordService.java
package com.ty.lab.service;

import com.ty.lab.entity.PageResult;
import com.ty.lab.pojo.TbLabrecord;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface LabrecordService {

    /**
     * 返回全部列表
     *
     * @return
     */
    public List<TbLabrecord> findAll();

    /**
     * 返回分页列表
     *
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize);

    /**
     * 增加
     */
    public void add(TbLabrecord labrecord);

    /**
     * 修改
     */
    public void update(TbLabrecord labrecord);

    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    public TbLabrecord findOne(Long id);

    /**
     * 批量删除
     *
     * @param ids
     */
    public void delete(Long[] ids);

    /**
     * 分页
     *
     * @param pageNum  当前页 码
     * @param pageSize 每页记录数
     * @return
     */
    public PageResult findPage(TbLabrecord labrecord, int pageNum, int pageSize);

}
